internal/driver/driver.go

```diff
@@ -56,6 +54,21 @@ func NewProtocolDriver() sdkModel.ProtocolDriver {
 	})
 	return driver
 }
+func createX509Template() x509.Certificate {
+	template := x509.Certificate{
+		SerialNumber: big.NewInt(2023),
```

Comment:
Is this always going to be fixed? This serial number?

Reply:
We would have to ask JP to be certain, I don't know

Reply:
I am letting it go for now, and wait for JP once he's there we can raise the point and ask him, for now, going ahead with the approval.